eessi_bot_job_manager.py

```diff
@@ -43,9 +43,8 @@
 
 # Local application imports (anything from EESSI/eessi-bot-software-layer)
 from connections import github
-from tools import config, run_cmd
+from tools import config, job_metadata, run_cmd
 from tools.args import job_manager_parse
-from tools.job_metadata import read_job_metadata_from_file, read_metadata_file
 from tools.pr_comments import get_submitted_job_comment, update_comment
 
 
```

Comment:
We also still refer to tarballs in the constants set

Reply:
Removed those constants in https://github.com/EESSI/eessi-bot-software-layer/pull/263/commits/34ead4dd9d8c728988db92aa22fd08cb6fb5c180 because they weren't used anywhere in the code.